airgun/views/contenthost.py

```diff
@@ -146,6 +147,12 @@ class details(SatTab):
         details = ReadOnlyEntry(name='Details')
         auto_attach = EditableEntryCheckbox(name='Auto-Attach')
         service_level = EditableEntrySelect(name='Service Level')
+        # System Purpose
+        system_purpose_status = ReadOnlyEntry(name='System Purpose Status')
+        service_level = EditableEntrySelect(name='Service Level (SLA)')
+        usage_type = EditableEntrySelect(name='Usage Type')
+        role = EditableEntrySelect(name='Role')
+        addons = EditableEntryMultiCheckbox(name='Add ons')
```

Comment:
can you provide screenshot for that area as I cannot reproduce the behavior? Is it list of checkboxes?

Reply:
![syspurpose](https://user-images.githubusercontent.com/6850667/52058677-3eaf1980-2568-11e9-8435-d663600cb007.png)